mint.json

```diff
@@ -57,9 +57,10 @@
     {
       "group": "Devices",
       "pages": [
-        "api-reference/endpoint/get",
+        "api-reference/endpoint/get-devices",
```

Comment:
Can you update the order to 

1. get-devices
2. get-device
3. edit
4. send-command

That way we start a pattern of grouping like commands together as the list of options grows!